src/Entity/Post.php

```diff
@@ -49,12 +49,12 @@ public function getId(): ?int
 
     public function getDogId(): ?dog
     {
-        return $this->dog_id;
+        return $this->dog;
     }
 
-    public function setDogId(?dog $dog_id): self
+    public function setDogId(?Dog $dog): self
     {
-        $this->dog_id = $dog_id;
+        $this->dog = $dog;
 
         return $this;
     }
```

Comment:
Shouldn't these be setting an ID? If they are setting Dog type fields on the Post entity, the name should reflect that. Although, I feel that the id is much more relevant to the Post

Reply:
Yeah, definitely. I quite tricky to find all the things that need changing, so left as it is and just fed it what waas defined.

Reply:
We'll refactor this tomorrow and talk more about what is happening here 👍